lib/calc.rb

```diff
@@ -1,3 +1,33 @@
 # frozen_string_literal: true
 
 # Calculator Class here
+class Calculator
+  attr accessor :starting_number
+  attr accessor :second_number
+  def initialize(starting_number)
+    @starting_number = starting_number
+  end
+
+  def addition
```

Comment:
The arithmetic methods should be named `add`, `subtract`, `multiply`, and `divide`.